plot/plot.app.js

```diff
@@ -683,14 +695,56 @@
+                                                                    values.legend_markdown_pattern.push(linkWithContextParams);
+                                                                    traceLabel=traceLabel.replace(extractedLink,linkWithContextParams)
+                                                                }catch(error){
+                                                                    traceLabel=row[trace.legend_col];
+                                                                }
+                                                                //delete $rootScope.templateParams;
+                                                            }
+                                                            values.labels.push(traceLabel);
+                                                            //Added the legend column names to text variable so that the tooltip name does not contain a link if the legends contains a link
+                                                            var tooltip=trace.hovertemplate_display_pattern ? ERMrest.renderHandlebarsTemplate(trace.hovertemplate_display_pattern, $rootScope.templateParams) : row[trace.legend_col];
+                                                            values.text.push(tooltip);
                                                         }
                                                         values.values.push(formatData(row[trace.data_col], plot.config ? plot.config.format_data : false, "pie"));
+                                                        if(trace.hasOwnProperty("graphic_link_pattern")){
+                                                            var qCharacter = trace.graphic_link_pattern.indexOf("?") !== -1 ? "&" : "?";
+                                                            let graphic_link=trace.graphic_link_pattern+qCharacter+"pcid="+contextUrlParams.cid+"&ppid="+contextUrlParams.pid;
+                                                            values.graphic_link_pattern.push(ERMrest.renderMarkdown(ERMrest.renderHandlebarsTemplate(graphic_link,$rootScope.templateParams),true));
+                                                        }
+                                                        delete $rootScope.templateParams.$self;
                                                     });
                                                     plot_values.data.push(values);
+                                                    if(plot.config.hasOwnProperty("title_display_markdown_pattern")){
+                                                        var link=/\((.*?)\)/ig.exec(plot.config.title_display_markdown_pattern)[1];
+                                                        var qCharacter = link.indexOf("?") !== -1 ? "&" : "?";
+                                                        $rootScope.templateParams.$layout=layout
```

Comment:
We aren't planning on supporting this use case.

Reply:
This is for extracting the title